topics/computational-chemistry/tutorials/md-simulation-gromacs/tutorial.md

```diff
@@ -0,0 +1,299 @@
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: computational-chemistry
+    tutorials:
+      - analysis-md-simulations
+contributors:
+  - simonbray
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+Molecular dynamics (MD) is a method to simulate molecular motion by iterative application of Newton's laws of motion. It is often applied to large biomolecules such as proteins or nucleic acids.
+
+Multiple packages exist for performing MD simulations. One of the most popular is the open-source GROMACS, which is the subject of this tutorial. Other MD packages which are also wrapped in Galaxy are [NAMD]({{ site.baseurl }}{% link topics/computational-chemistry/tutorials/md-simulation-namd/tutorial.md %}) and CHARMM (available in the [docker container](https://github.com/scientificomputing/BRIDGE)).
+
+This is a introductory guide to using GROMACS {% cite abraham15 %} in Galaxy to prepare and perform molecular dynamics on a small protein. For the tutorial, we will perform our simulations on hen egg white lysozyme. 
+
+> ### {% icon comment %} More information
+> This guide is based on the GROMACS tutorial provided by Justin Lemkul [here](http://www.mdtutorials.com/gmx/lysozyme/index.html) - please consult it if you are interested in a more detailed, technical guide to GROMACS. 
+{: .comment}
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+
+# Process
+
+Prior to performing simulations, a number of preparatory steps need to be executed.
+
+The process can be divided into multiple stages:
+ 1. Setup (loading data, solvation i.e. addition of water and ions)
+ 2. Energy minimization of the protein
+ 3. Equilibration of the solvent around the protein (with two ensembles, NVT and NPT)
+ 4. Production simulation, which produces our trajectory.
+
+The trajectory is a binary file that records the atomic coordinates at multiple time steps, and therefore shows the dynamic motion of the molecule. Using visualization software, we can display this trajectory as a film displaying the molecular motion of the protein. We will discuss each step making up this workflow in more detail.
+
+
+# Getting data
+To perform simulation, an initial PDB file is required. This should be 'cleaned' of solvent and any other non-protein atoms. Solvent will be re-added in a subsequent step.
+
+A prepared file is available via Zenodo. Alternatively, you can prepare the file yourself. Download a PDB structure file from the [Protein Data Bank](https://www.rcsb.org/) and remove the unwanted atoms using the grep text processing tool. This simply removes the lines in the PDB file that refer to the unwanted atoms.
+
+
+> ### {% icon hands_on %} Hands-on: Upload an initial structure
+> 1. First of all, create a new history and give it a name.
+>
+>    {% include snippets/create_new_history.md %}
+>
+> 2. Upload the file in Galaxy from the PDB:
+>    >    ```
```

Comment:
You can remove the extra `>` here before the quotes (same for the next 3 lines). It will make the link copy-able.